activesupport/lib/active_support/cache/mem_cache_store.rb

```diff
@@ -199,10 +187,15 @@ def deserialize_entry(raw_value)
           if raw_value
             entry = Marshal.load(raw_value) rescue raw_value
             entry.is_a?(Entry) ? entry : Entry.new(entry)
-          else
-            nil
           end
         end
+
+        def rescue_error(fallback)
```

Comment:
When passing in false or nil, it feels like you're turning something off. Mitigate by making this a keyword argument with `_to` suffix (since ´fallback: false` still sounds like turning something off):

```ruby
def rescue_error(fallback_to: nil)
  # ...
end
```

(Note, I think we can safely default it to nil)